Fix path to libtool scripts so autoreconf can succeed.

Upstream-Status: Inappropriate [Other]
Workaround is specific to our build system.

Signed-off-by: Scott Garman <scott.a.garman@intel.com>

diff -urN sudo-1.7.4p4.orig//aclocal.m4 sudo-1.7.4p4//aclocal.m4
--- sudo-1.7.4p4.orig//aclocal.m4	2010-08-06 07:17:36.000000000 -0700
+++ sudo-1.7.4p4//aclocal.m4	2010-11-22 14:38:20.360957038 -0800
@@ -368,8 +368,8 @@
 dnl
 dnl Pull in libtool macros
 dnl
-m4_include([libtool.m4])
-m4_include([ltoptions.m4])
-m4_include([ltsugar.m4])
-m4_include([ltversion.m4])
-m4_include([lt~obsolete.m4])
+m4_include([m4/libtool.m4])
+m4_include([m4/ltoptions.m4])
+m4_include([m4/ltsugar.m4])
+m4_include([m4/ltversion.m4])
+m4_include([m4/lt~obsolete.m4])
